ci/conda_env_cpp.yml

```diff
@@ -20,7 +20,6 @@
 boost-cpp=1.68.0
 brotli
 bzip2
-cmake
```

Comment:
Use system cmake instead of the latest one available in conda. This will help to maintain compatibility with older cmakes.